scripts/jenkins-yoctobuild-build.sh

```diff
@@ -8,6 +8,15 @@ echo "Debug Jenkins setup"
 ls /home/jenkins/.ssh
 #----
 
+disable_mender_service() {
+    if [ "$DISABLE_MENDER_SERVICE" = "true" ]
+    then
+        cat >> "$BUILDDIR"/conf/local.conf <<EOF
+SYSTEMD_AUTO_ENABLE = "disable"
```

Comment:
It should be `SYSTEMD_AUTO_ENABLE_pn-mender = "disable"`, otherwise you disable *all* services.